Updated Transitioning to stencil per Style Guide (#518)
bigcommerce/dev-docs: docs/stencil-docs/getting-started/transitioning-to-stencil.md

### On This Page
- [What is Stencil?](#what-is-stencil)
- [Why transition to Stencil?](#why-transition-to-stencil?)
- [Technical differences](#technical-differences)
  - [Developing locally on Stencil CLI](#developing-locally-on-stencil-cli)

>   - [Using handlebars](#using-handlebars)

Reviewer: ```suggestion
  - [Using Handlebars](#using-handlebars)

```
Handlebars should be capitalized

Stencil themes are built on non-proprietary, open-source code to provide easier adoption and adaptation while providing greater flexibility and access for customization. In comparison, Blueprint is built on a rigid, proprietary codebase, making it difficult to tailor and extend theme functionality.

For more information on the enhanced capabilities of Stencil, see the [Stencil vs. Legacy Blueprint](https://support.bigcommerce.com/s/article/The-Stencil-Theme-Platform#compare) page.

## Technical differences

When transitioning any store from Blueprint to Stencil, you should be aware of a few key differences. Transitioning to Stencil allows developers to do the following:
- [Develop locally on Stencil CLI](#developing-locally-on-stencil-cli)
- [Use Handlebars instead of global variables](#use-handlebars)
- [Customize Store Design mode](#customize-store-design-mode)
- [Create custom templates](#create-custom-templates)

### Developing locally on Stencil CLI

> For complete control over a theme’s appearance, logic, and theme configuration files, developers can use the Stencil Command Line Interface (Stencil CLI). This interface allows you to design and preview storefronts locally before pushing them to production. Access to real-time previews and testing across various devices is available via [Browsersync](https://www.browsersync.io/), a browser testing assistant built into Stencil CLI.

Reviewer: ```suggestion
For complete control over a theme’s appearance, logic, and theme configuration files, developers can use the Stencil command line interface (Stencil CLI). This interface allows you to design and preview storefronts locally before pushing them to production. Access to real-time previews and testing across various devices is available via [Browsersync](https://www.browsersync.io/), a browser testing assistant built into Stencil CLI.

```

While Blueprint themes allow users to add new files and assets to themes via WebDav, Stencil assets must be added to and bundled into the theme, or referenced from an external source.

For images referenced from anywhere in your Blueprint file directory, you will need to move these assets to your Stencil theme and reference them appropriately. You can still reference assets from within the `/content/` folder in WebDav or from an external source.

For more information on Stencil theme assets, see the [Theme Assets](https://developer.bigcommerce.com/stencil-docs/storefront-customization/theme-assets) page.

> ### Using handlebars

Reviewer: ```suggestion
### Using Handlebars

```

While PHP variables enclosed within `%%` markers represent dynamic content, [Handlebars](https://handlebarsjs.com/), a JavaScript templating language represents BigCommerce dynamic stored data in Stencil themes.

> Handlebars keep your HTML page clean by separating the logic-less templates from the business logic in your JavaScript files. This organization improves the structure of the application, promoting maintainability and scalability. Handlebars also simplify the task of manually updating data in the customer-facing display view.

Reviewer: ```suggestion
Handlebars keeps your HTML page clean by separating the logic-less templates from the business logic in your JavaScript files. This organization improves the structure of the application, promoting maintainability and scalability. Handlebars also simplifies the task of manually updating data in the customer-facing display view.

```

Author: Is Handlebars singular or plural?
